### ISSUE TITLE
2S - 3

### ISSUE THREAD

**Issue by Dfinitski** [Mi-Walkie-Talkie-by-Darkhorse/Mi-Walkie-Talkie-Plus#10]
Hi
Mi walkie talkie 2s and 3 are not supported. No connecting.
Could you add that support please ?

**Comment by swas** [comment]
There are a new chinese app version: 2.17.21
On this version the 2s works, but its on chinese :(

**Comment by Dfinitski** [comment]
Hi. Here is a n app in english
https://drive.google.com/file/d/1lcbyujakjlT-ApI-hN5EOTU_hdfqEm8j/view?usp=sharing
It works as mi 2.

**Comment by Dfinitski** [comment]
2.17.21 wich works on my smartphone in english
https://drive.google.com/file/d/1FjvNzplb5yQrRHV2kPUssIy89XFdStSx/view?usp=sharing

**Comment by Dfinitski** [comment]
It works with limited frequency range, that is the problem

**Comment by swas** [comment]
Thanks @Dfinitski !

You are right, my andoid device language is spanish. If I configure english, I can see now the app (2.17.21) in english.

**Comment by Dimahay** [comment]
Did you find any solutions for this?

**Comment by swas** [comment]
No @Dimahay 
only wait for the developer :(

**Comment by foxenonx** [comment]
Now 2024 at present there is no custom firmware for Xiaomi Walkie Talkie 3 to config frequency morethan 430-440Mhz but I need 400-440Mhz Thank 😣

**Comment by rooyblack** [comment]
> Now 2024 at present there is no custom firmware for Xiaomi Walkie Talkie 3 to config frequency morethan 430-440Mhz but I need 400-440Mhz Thank 😣
What find answer Now 2025? Please review

**Comment by Dimahay** [comment]
I was able to set custom channels in PMR 446 on a Xiaomi Walkie Talkie 3 by installing the app on an old android device setted in Chinese language. For some reason other languages didn't work. You should set the language in the device settings.

**Comment by rooyblack** [comment]
Can you tell me what the application (series) is? And the old Android, how much Android? Tx so much

**Comment by Dimahay** [comment]
i think every android version is alright. I don't remember the version of the apk, you should give a try with the last one. Unfortunatelly I don't have that phone with me. The only thing I remember is that I had to set the phone in chinese or english and the app was in Chinese

**Comment by rooyblack** [comment]
Yes, can you remember when it was done? Considering that currently there is
the latest firmware

Pada Rab, 19 Feb 2025 17.11, DimaHay ***@***.***> menulis:

**Comment by Dimahay** [comment]
I did it approximately 5 months ago so probably with the last version

**Comment by rooyblack** [comment]
Tx, one again? Is the APK used by Mi Home or Mi PTT+ in circulation? Tx

**Comment by Dfinitski** [comment]
No, you will need a custom firmware because FW has a limitations too.

**Comment by Dimahay** [comment]
the UI of the app was the same of the Darkhorse version. 
Anyways my walkie was 2S. Not 3. sorry for the confusion.
Not the LTE version
